qemu/tests/ioeventfd.py

```diff
@@ -0,0 +1,163 @@
+"""
+ioeventfd.py include following case:
+    1. Test ioeventfd under stress.
+    2. Check the ioeventfd property.
+"""
+import logging
+import re
+
+from virttest import error_context
+from virttest import env_process
+from virttest import utils_test
+from virttest import qemu_qtree
+from virttest import utils_misc
+from avocado.utils import process
+from provider.storage_benchmark import generate_instance
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test the related functions of ioeventfd.
+    Step:
+        Scenario 1:
+            1.1 Start guest with ioeventfd=off.
+                For windows: Check whether vioscsi.sys verifier enabled in
+                the guest.
+            1.2 Dd file in the guest.
+                For linux: dd if=/dev/zero of=/home/test bs=1M count=4096
+                oflag=direct.
+                For windows: We can load some big data to the disk via
+                DummyCMD.exe tools or other tools.
+            1.3 Reboot the guest.
+            1.4 Run iozone in the guest.
+            1.5 Power off the guest.
+            1.6 Repeat the step 1.1-1.5 with ioeventfd=on.
+        Scenario 2:
+            2.1 Boot guest with ioeventfd=off.
+            2.2 Execute info qtree in QMP monitor, info qtree should show the
+                ioeventfd = false.
+            2.3 Check the ioeventfd=off via /proc/$PID/fd/.
+            2.4 Boot guest with ioeventfd=on.
+            2.5 Execute info qtree in QMP monitor, info qtree should show the
+                ioeventfd = true.
+            2.6 Check the ioeventfd=on via /proc/$PID/fd/.
+            2.7 Compare the output of 'ls -l /proc/$PID/fd/', the fds with
+                "off" should be less than the one with "on".
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    def _set_ioeventfd_options():
```

Comment:
all these functions are sub-functions of def run(test, params, env), and py file for test case are not allowed importe to others test case, so I think no need set private label for them here~

Reply:
My purpose is not allowed by other cases.